source/find_getmore_killcursors_commands.rst

```diff
@@ -294,23 +309,56 @@ when sent to a secondary. The OP_QUERY namespace MUST be the same as for the
 
 .. _getMore: https://www.mongodb.com/docs/manual/reference/command/getMore/
 
+.. code:: typescript
+
+  interface GetMoreCommand {
+    /** Set to the nonzero cursor id */
+    getMore: int64;
+    /** Set to the namespace returned by the initial command response */
+    collection: string;
+    /**
+     * User configurable document count for the batch returned for this getMore.
+     * Only attached to command document if nonzero.
+     */
+    batchSize?: int32;
+    /**
+     * User configurable time limit enforced by the server.
+     */
+    maxTimeMS?: int32;
+    /**
+     * User configurable comment that can be used to identify the operation in logs.
+     * This can be any BSON value.
+     */
+    comment?: BSONValue;
+  }
+
 The **batchSize** option of **getMore** command MUST be an int32 larger than 0. If **batchSize** is equal to 0 it must be omitted. If **batchSize** is less than 0 it must be turned into a positive integer using **Math.abs** or equivalent function in your language.
 
 On success, the getMore command will return the following:
 
-.. code:: javascript
-
-    {
-      "cursor": {
-        "id": <int64>,
-        "ns": <string>,
-        "nextBatch": [
-          ...
-        ]
-      },
-      "ok": 1
+.. code:: typescript
+
+    interface GetMoreResponse {
+      ok: 1;
+      cursor: {
+        /** The cursor id, this may be equal to zero indicating the cursor is exhausted */
```

Comment:
I'm not entirely familiar with each field in the [killCursors](https://www.mongodb.com/docs/manual/reference/command/killCursors/) response. My only experience is relying on `cursorsKilled` to tell us what was actually killed by the command.

Since the context here is a cursor that was once alive, I think the universal term for a cursor whose ID is now reported as zero is "dead". I know some drivers have historically provided boolean methods on the cursor with those names (e.g. `isAlive()`, `isDead()`).

To me, "killed" suggests some preemptive action (e.g. session terminated or explicitly calling `killCursors`), and "exhausted" suggests a more natural death by completing iteration of a non-tailable cursor. "Closed" just seemed like a nicer alternative to saying "dead" here.

"Exhausted or ___" is sufficient for me, though, so I'll spare you the additional bike-shedding. Feel free to resolve on your own.